### PR TITLE
More Japanese Brands (part 4)

### PR DESCRIPTION
**Pull request by sguinetti**
This pull request adds six new brands in Japan (to reduce the workload in this request, I will add others in the future). With Wikipedia, Wikidata info and english name. In the articles that I found in this encyclopedia they say that the companies have several headquarters of which they are mapped in Openstreetmap.

### CODE REVIEW

**Comment by Adamant36** [comment]
Hi there. What's with alt_name for ニューデイズ? Is there a reason it can't just be name:jp?

**Comment by sguinetti** [comment]
The japanese name is in alt_name key because is not a main name. In [FAQ page](https://www.j-retail.jp/contact/faq/detail.html?id=1&category_id=2) and [Twitter username](https://twitter.com/newdays_jp) says "NewDays" many times (almost every time). "ニューデイズ" secondary name is used in some external websites (as it is shown in [Wikpedia](https://ja.wikipedia.org/wiki/NewDays)).

File: brands/amenity/vending_machine.json [hunk 1]
```diff
@@ -168,5 +168,47 @@
...
+      "brand:ja": "コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Nippon Coca Cola Co., Ltd.",
+      "operator:wikidata": "Q11506438",
+      "operator:wikipedia": "ja:日本コカ・コーラ",
+      "vending": "drinks",
+      "wikidata": "Q2813",
```
Comment (bhousel) [inline brands/amenity/vending_machine.json]: `brand:wikidata`

File: brands/amenity/vending_machine.json [hunk 2]
```diff
@@ -168,5 +168,47 @@
...
+      "wikipedia": "ja:コカ・コーラ"
```
Comment (bhousel) [inline brands/amenity/vending_machine.json]: `brand:wikipedia`

File: brands/amenity/vending_machine.json [hunk 3]
```diff
@@ -168,5 +168,47 @@
...
+    "tags": {
+      "amenity": "vending_machine",
+      "brand": "コカ・コーラ",
+      "brand:en": "Coca-Cola",
+      "brand:ja": "コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Nippon Coca Cola Co., Ltd.",
```
Comment (bhousel) [inline brands/amenity/vending_machine.json]: Is Coca-Cola Japan really the operator of all the vending machines?

File: brands/amenity/vending_machine.json [hunk 4]
```diff
@@ -168,5 +168,47 @@
...
+      "brand": "ポッカサッポロ",
+      "brand:en": "Pokka Sapporo",
+      "brand:ja": "ポッカサッポロ",
+      "brand:wikidata": "Q7208665",
+      "brand:wikipedia": "ja:Q7208665",
+      "name": "ポッカサッポロ",
+      "name:en": "Pokka Sapporo",
+      "name:ja": "ポッカサッポロ",
+      "operator": "ポッカサッポロフード&ビバレッジ",
+      "operator:en": "Pokka Sapporo Food & Beverage",
```
Comment (bhousel) [inline brands/amenity/vending_machine.json]: If this really is the operator of all the vending machines, you can add `operator:wikidata` also

File: brands/amenity/vending_machine.json (hunk 3 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: Yes. I found from [this PDF](https://www.cocacola.co.jp/content/dam/journey/jp/ja/private/2017/10/factsheet/Coca-Cola-Japan-System-Fact-Sheet_2017Q4_EN.PDF).

File: brands/amenity/vending_machine.json (hunk 4 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: I deleted the operator beacuse the wikipedia article is the same what the brand:wikipedia. I do not know if it is a right decision.

File: brands/amenity/vending_machine.json (hunk 2 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: Fixed

File: brands/amenity/vending_machine.json (hunk 1 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: Fixed

**Comment by sguinetti** [comment]
One question, would it be right to share the same article on the two keys? I would add the Pokka Sappor article in "brand:wikipedia" and "operator:wikipedia". This issue isn't defined in CONTRIBUTING.md.

File: brands/shop/convenience.json [hunk 5]
```diff
@@ -916,6 +916,22 @@
...
+      "shop/convenience|NewDays ミニ",
+      "shop/convenience|ニューデイズ"
+    ],
+    "nocount": true,
+    "tags": {
+      "alt_name": "ニューデイズ",
+      "brand": "NewDays",
+      "brand:wikidata": "Q11234763",
+      "brand:wikipedia": "ja:NewDays",
+      "name": "NewDays",
```
Comment (matkoniecz) [inline brands/shop/convenience.json]: Is it actually using (in Japan) name NewDays rather than ニューデイズ?

File: brands/amenity/vending_machine.json [hunk 6]
```diff
@@ -168,5 +168,45 @@
...
+      "brand": "コカ・コーラ",
+      "brand:en": "Coca-Cola",
+      "brand:ja": "コカ・コーラ",
+      "brand:wikidata": "Q2813",
+      "brand:wikipedia": "ja:コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Coca Cola Japan Co., Ltd.",
```
Comment (matkoniecz) [inline brands/amenity/vending_machine.json]: I am unsure about this one. Are you sure that every single vending machine dispensing Coca-Cola in Japan is operated by Coca Cola Japan Co., Ltd.?

**Comment by matkoniecz** [comment]
> One question, would it be right to share the same article on the two keys? I would add the Pokka Sappor article in "brand:wikipedia" and "operator:wikipedia". This issue isn't defined in CONTRIBUTING.md.

This sis perfectly fine, though I have doubts about operator tag itself.

**Comment by matkoniecz** [comment]
>  In FAQ page and Twitter username says "NewDays" many times (almost every time).

What about its shops? Is it also using "NewDays" rather than some Japanese name on signs etc?

File: brands/shop/convenience.json (hunk 5 above)
Comment (sguinetti) [reply brands/shop/convenience.json]: Yes. I search in Youtube NewDays and this name is official.

File: brands/amenity/vending_machine.json (hunk 6 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: I my opinion, yes. We could ask a third party to confirm. Right?

**Comment by sguinetti** [comment]
> > In FAQ page and Twitter username says "NewDays" many times (almost every time).
> 
> What about its shops? Is it also using "NewDays" rather than some Japanese name on signs etc?

Similar to Youtube, in Twitter this name is in english. Even if the account is directed to the Japanese public.

File: brands/shop/convenience.json (hunk 5 above)
Comment (matkoniecz) [reply brands/shop/convenience.json]: In that case I will mark this one as resolved.

File: brands/amenity/vending_machine.json (hunk 6 above)
Comment (sguinetti) [reply brands/amenity/vending_machine.json]: `The Coca-Cola system in Japan is comprised of Coca-Cola (Japan) Co, Ltd., which supplies concentrates, plans and develops new products, and conducts its marketing and advertising activities; system bottling partners, **which manufacture and sell products**, and other affiliated companies.  Coca-Cola (Japan) Co, Ltd. was established in 1957 as the Japanese subsidiary of The Coca-Cola Company`
If you mean that the store or market sells the product, then I could give you a reason. However, it is the company "provides the services" to its partners. That's why I tell you that it is Coca-Cola Japan as a main operator.

Comment (sguinetti) [reply brands/amenity/vending_machine.json]: Other quote about vending machines from [Coca-Cola website](http://j.cocacola.co.jp/peak_shift/about/):

> The development and rapid development of the peak shift vending machine of Coca-Cola, and unique public relations activities were evaluated, and the 2013 Energy Conservation Grand Prize (Energy Conservation Case Category) Economy and Industry Minister's Prize (Power Conservation Award) was received!
That means that Coca-Cola manufactures, sells and improves its machines. Even the company is plaing that all its machines are of low energy consumption.

Comment (sguinetti) [reply brands/amenity/vending_machine.json]: I think we're already sure who the operator is. I hope I'm not wrong

**Comment by bhousel** [comment]
Thanks @sguinetti I just merged it. 
If the `operator` thing turns out to be wrong on the Coca-Cola vending machines, someone can just remove them later.  It's really not a big deal.

**Pull request merged**